gradle-baseline-java-config/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -19,6 +19,7 @@
     <suppress files="\.(bdr|eot|gif|gzip|jar|json|otf|png|svg|ttf|woff|zip)$" checks="NewlineAtEndOfFile" />
 
     <!-- Generated code should not be subjected to checkstyle. -->
+    <suppress files="[/\\].*[/\\]generated[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
```

Comment:
`build/generated/*` seems better than `{projectDir}/generated_src/`. my only concern is how that feels when working on things from an IDE (since `build/` is not an intellij/eclipse concept). i assume it should be fine, but curious to how it affects the developer experience - since functionally all 3 options are equivalent.

my vote: having generated code go in `src/generated/` gives enough signal that the code is generated and not to be modified. other pros: it lives next to my other source code and it's easy to find in my IDE. also flagging the directory as "Generated Source Code" in intellij makes it even clearer.